js/gridfieldlinkbutton.js

```diff
@@ -0,0 +1,10 @@
+(function ($) {
+    $.entwine('ss', function ($) {
+        $('.ss-gridfield .gridfield-button-link').entwine({
+            onclick: function (e) {
+                // Overrides entwine preventing the default, thus ensures the link is opened on
+                // button click.
```

Comment:
Ok this is a little clearer now, but I don't know why we should need to have this to make an anchor element work the way it's meant to by default. Are you able to provide some more context around this, like where entwine is preventing the default from?